splat.js

```diff
@@ -1,18 +1,106 @@
 'use strict';
 
-const format = require('./format');
 const util = require('util');
+const { SPLAT } = require('triple-beam');
+
+/**
+ * Captures the number of format (i.e. %s strings) in a given string.
+ * Based on `util.format`, see Node.js source:
+ * https://github.com/nodejs/node/blob/b1c8f15c5f169e021f7c46eb7b219de95fe97603/lib/util.js#L201-L230
+ * @type {RegExp}
+ */
+const formatRegExp = /%[sdjifoO%]/g;
```

Comment:
https://developers.google.com/web/tools/chrome-devtools/console/console-write depending how far you want to go with support, you might want to add `%c` in this list as well as it's used for color in browser `console.log` statements.